modules/executive/components/ExecutiveOverviewCard.tsx

```diff
@@ -29,14 +29,12 @@ type Props = {
 
 export default function ExecutiveOverviewCard({ proposal, isHat, spellData, ...props }: Props): JSX.Element {
   const { trackButtonClick } = useAnalytics(ANALYTICS_PAGES.EXECUTIVE);
-  const account = useAccountsStore(state => state.currentAccount);
+  const { account } = useAccount();
   const { data: mkrOnHat } = useMkrOnHat();
   const [voting, setVoting] = useState(false);
   const { data: votedProposals } = useVotedProposals();
-  const network = getNetwork();
   const bpi = useBreakpointIndex();
   const { comments } = useExecutiveComments(proposal.address);
-  const canVote = !!account;
```

Comment:
I liked this because if the `canVote` logic changes, it just needs to be updated in one spot.

Reply:
yes? mh ok

Reply:
i think !account is something that is more or less extended through the app ( and in general on web3 apps),

Reply:
restored